### PR TITLE
blurred background and removal of border for video and slides

### PR DESCRIPTION
**Pull request by shubham-tyagi**
Signed-off-by: shubham-tyagi <raj.tyagi2000@gmail.com>

**Description**
As discussed in the video, the background is blurred and the border for the modal is removed. I have made some really small modifications apart from that.

This PR fixes #

**Notes for Reviewers**
Please point out if I missed something or if anything else is needed to be done. I could remove those small modifications if they are not looking good.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: collections/_pages/talks-and-presentations.html [hunk 1]
```diff
@@ -57,14 +57,15 @@
 }
 iframe{
   margin: 0 2rem 0 2rem;
-}
-.modal-window > div {
   border-radius: 1rem;
```
Comment (debo19) [inline collections/_pages/talks-and-presentations.html:60]: I don't think we should apply curved borders to the video.

**Review by debo19 — COMMENTED** [review]
Other changes look good.

File: collections/_pages/talks-and-presentations.html (hunk 1 above)
Comment (vedant-kakde) [reply collections/_pages/talks-and-presentations.html:60]: +1

Comment (shubham-tyagi) [reply collections/_pages/talks-and-presentations.html:60]: okay will revert that, does it not look good or some design violations?

Comment (debo19) [reply collections/_pages/talks-and-presentations.html:60]: @shubham-tyagi no not any violation, just looks a little blunt to me, you can try reducing the border-radius a bit  ( 0.5 rem maybe ), that should work too.

Comment (shubham-tyagi) [reply collections/_pages/talks-and-presentations.html:60]: yeah sure, will do that.

**Review by debo19 — COMMENTED** [review]
This button should be center-aligned.

button

**Comment by shubham-tyagi** [comment]
Hey @debo19 , I will do the button center aligned , for now can you review my recent commit. 

- I have removed the image link so that even if we click on the image it font gets enlarged
- I have also added a functionality that if we click outside of the overlay(not only on the cross) then too the overlay will vanish.

**Comment by debo19** [comment]
@shubham-tyagi The button isn't working for me in the preview, the modal isn't visible. Is it working fine for you?

**Comment by shubham-tyagi** [comment]
@debo19 In my local machine, it's working fine. What might be the problem here?

**Comment by shubham-tyagi** [comment]
I guess Its not working because the JS file is not included in the preview(I am just guessing). Because in my local machine as you can see in the video it's working fine. @warunicorn19 thoughts here ?

ezgif com-gif-maker

**Comment by shubham-tyagi** [comment]
Hey @debo19 , It's working fine in the preview now. Please see if its working fine in your machine.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
@shubham-tyagi really like the javascript approach😅, couple of nitpicks, could you please center align the button and the <script> tag that you mentioned at the end of the page, could you please put that right below the <style> tag (good practice to maintain that format).

Good work💪.

**Comment by shubham-tyagi** [comment]
@warunicorn19 when I include script tag just after style tag then it reverses the page. And I am not sure but isn't it best to include js script after the page has been loaded? What do you think should be done @debo19 @warunicorn19

**Comment by warunicorn19** [comment]
huh🤔? i guess what you're saying makes sense because html loads from top to bottom. okay, ignore what i said about the <script> tag. aligning the button to the center will do

**Comment by shubham-tyagi** [comment]
@warunicorn19  Could you please check out the screenshot I attached in my previous comment, Should it full-page centered or this is fine?

**Comment by warunicorn19** [comment]
Yup the position in the screenshot works👌

File: collections/_pages/talks-and-presentations.html [hunk 2]
```diff
@@ -102,34 +101,33 @@
...
-    <div class="text ">
+    <div class="text " style="text-align: justify; text-justify: inter-word">
       <h2>CNCF On-Demand Webinar: <strong>Meshery - The Service Mesh Manager</strong></h2>
       <p> This webinar introduces Meshery, an open-source, multi-service mesh management plane that can provision nine different service meshes, their sample applications and benchmarks the performance of service mesh deployments.
         <br>
-      <div align="center" class="button-para" style="margin-top: 3rem;">
-        <a href="#open-modal" class="link">Find video and slides here</a>  
-      </div>
+        <div class="button-para" style="margin-top: 3rem; ">
+          <button class="link" style="position: relative;left: 25%;">Find video and slides here</button>
```
Comment (debo19) [inline collections/_pages/talks-and-presentations.html:109]: mobile2
The button should be aligned to the center in mobile view too.  Also, can you remove the inline styles and add them inside the style tag.

**Review by debo19 — COMMENTED** [review]
Great work here @shubham-tyagi 🚀 . This should be the last change 😅

**Review by warunicorn19 — APPROVED** [review]
LGTM! Thanks @shubham-tyagi

**Pull request merged**